### ISSUE TITLE
coco eval perfomance

### ISSUE THREAD

**Issue by hao522** [roytseng-tw/Detectron.pytorch#1]
Excellent work! Have you trained from scratch and how's the performance on COCO evaluation? BTW, could you share some pre-trained weights to test on? Thanks a lot!

**Comment by yuzcccc** [comment]
Same question. 
Besides, how is the GPU memory occupation compared to the original implementation based on caffe2?

**Comment by roytseng-tw** [comment]
Hello, sorry for the late reply.
**Q1: How is the train-from-scratch performance**
    There were some bugs in the computation of loss. I just fixed them and put it on training. Once I have some promising results, I'll update the README.  However, I don't have many gpus, so it takes longer to  converge QQ.

**Q2: How is the gpu memory consumption and speed?**
For training **e2e_mask_rcnn-R-50-C4** on 2 1080ti with batch-size 4 (2 per gpu), it costs about 10000 MB memory per gpu, and it takes about 120 ~ 140 sec every one hundred iterations.
I have also tried training the same network using Detectron, and I could only have 1 batch per gpu (7XXX MB). It takes around 0.5 sec per iteration for batch size 2 on 2 1080ti.
In short, it seems that my implementation takes less gpu memory and has comparable training speed.

**Comment by yuzcccc** [comment]
Thanks for the explanation, good job dude : )

**Comment by jwyang** [comment]
@roytseng-tw great work! How is the training going? I am wondering whether you've got the evaluation numbers on coco now, :)

**Comment by Rizhiy** [comment]
I have trained R-101-FPN and only got 0.33 on coco, more details in #24.

**Comment by roytseng-tw** [comment]
Hello everyone, I have update some benchmark results. Though I can't use same batch size (ie. 8 or 16) as in the config file to train the model, the trend of loss curves are quiet consistent.

**Comment by wmmxk** [comment]
Hi roytseng-tw. Thanks for sharing your code and results. Is it possible to share a checkpoint for me to learn your code?  Thanks.
